README.md

```diff
@@ -272,6 +272,40 @@ EOF
 # systemd-nspawn -bi image.raw
 ```
 
+## Create a debian image running under debian
+
+Inside an empty directory run (being root)
+
+```bash
+# apt-get install mkosi
+# mkdir mkosi.cache
+# cat > mkosi.default <<EOF
+# Just enough to use mkosi in it
+[Distribution]
+Distribution=debian
+Release=stretch
+
+[Output]
+Format=raw_gpt
+Bootable=yes
+
+[Packages]
+Packages=
+    git
+    mkosi
+EOF
+# cat > mkosi.postinst <<EOF
+# #!/bin/bash
+echo root:topSecret | chpasswd 
+EOF
```

Comment:
mkosi has direct support for setting the password, so please don't recommend doing it "manually" like this.